### DISCUSSION TITLE
Reduce Achievement Badge Size

### DISCUSSION THREAD

**Discussion by refcell** [community/community#20456] [Other Feature Feedback, Questions, & Ideas]
The achievement badge sizes seem way too large and out of proportion with respect to the rest of the profile, especially the size of organization pfps.

Concrete Proposal: Reduce badge size to that of the "organization" section images.

**Comment by mrcsxsiq** [comment]
I agree. The badges are too much larger than organization icons

**Comment by Loque18** [comment]
I think it would be better to re organize them, they should be the last item, and decrease a bit the size just a bit, here is a preview 
![image](https://user-images.githubusercontent.com/28642447/178826109-805af6d3-6605-4c87-80b6-676fa2ff021d.png)

**Reply by tvnr** [reply]
it really looks a lot better this way

**Reply by ThomasThelen** [reply]
100% agree. This is a huge improvement; IMO Organizations should be before Achievements

**Comment by rasitds** [comment]
This discussion should be considered.
